src/compiler/compile_rust_code.ts

```diff
@@ -37,3 +50,15 @@ export function compileRustCode(
         { stdio }
     );
 }
+
+function compileRustCodeNatively(canisterName: string, stdio: IOType) {
+    execSync(
+        `CARGO_TARGET_DIR=target cargo build --target wasm32-wasi --manifest-path .azle/${canisterName}/canister/Cargo.toml --release`,
```

Comment:
We aren't `cd`ing or `cp`ing any more... what's up with that?

Reply:
I can tell you